python/gsw/main.py

```diff
@@ -0,0 +1,370 @@
+                # Print whether or not indexing was successful
+                print("Iridium Report Status: "+iridium_res['result']+"\n\n")
+
+            elif self.recieved_uplink_confirmation:
+                # Create an iridium report and add that to the iridium_report index in elasticsearch. 
+                ir_report=json.dumps({
+                    "momsn":self.momsn,
+                    "mtmsn":self.mtmsn, 
+                    "confirmation-mtmsn": self.confirmation_mtmsn,
+                    "send-uplinks": self.send_uplinks,
+                    "time": str(datetime.now().isoformat())
+                })
+
+                # Index iridium report in elasticsearch
+                iridium_res = self.es.index(index='iridium_report_'+str(self.imei), doc_type='report', body=ir_report)
+                # Print whether or not indexing was successful
+                print("Iridium Report Status: "+iridium_res['result']+"\n\n")
+                # Record that we have not recently recieved any uplinks as we just indexed the most recent one
+                self.recieved_uplink_confirmation=False
+
+            time.sleep(1)
+
+    def disconnect(self):
+        '''
+        Stops the thread which is checking emails
+        and posting reports to elasticsearch
+        '''
+        self.run_email_thread=False
+        self.check_email_thread.join()
+
+# Get keys for connecting to email account and elasticsearch server
+try:
+    with open(os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '../usb_console/configs/radio_keys.json')))as radio_keys_config_file:
+        radio_keys_config = json.load(radio_keys_config_file)
+    with open(os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '../usb_console/configs/server_keys.json')))as server_keys_config_file:
+        server_keys_config = json.load(server_keys_config_file)
+except json.JSONDecodeError:
+    print("Could not load config files. Exiting.")
+    raise SystemExit
+except KeyError:
+    print("Malformed config file. Exiting.")
+    raise SystemExit
+
+# Create a read_iridium object
+readIr=read_iridium(radio_keys_config, server_keys_config)
+# Start checking emails and posting reports
+readIr.connect()
+
+# Set up the SwaggerUI API
+swagger_config={
+    "headers":[],
+    "specs":[
+        {
+            "endpoint":"apispec_1",
+            "route":"/apispec_1.json",
+            "rule_filter":lambda rule:True,
+            "model_filter":lambda tag:True
+        }
+    ],
+    "static_url_path": "/flassger_static",
+    "swagger_ui":True,
+    "specs_route":"/swagger/"
+}
+swagger=Swagger(app, config=swagger_config)
+
+# Endpoint for indexing data in elasticsearch
+# Mostly for testing purposes. We don't use this to actually index data in elasticsearch
+@app.route("/telemetry", methods=["POST"])
+@swag_from("endpoint_configs/telemetry_config.yml")
+def index_sf_report():
```

Comment:
It's not possible to add to a particular radio's data because this endpoint doesn't take IMEI number into account. This impeded testing for me. Can you add that in?